internal/config/config.go

```diff
@@ -64,8 +64,9 @@ type Config struct {
 }
 
 type Database struct {
-	URI     string `json:"uri" yaml:"uri"`
-	Managed bool   `json:"managed" yaml:"managed"`
+	URI      string `json:"uri" yaml:"uri"`
+	Managed  bool   `json:"managed" yaml:"managed"`
+	Analyzer *bool  `json:"analyzer" yaml:"analyzer"`
```

Comment:
We may regret putting this here? I think we should discuss for a min.

Reply:
```yaml
analyzer:
  database: false
```

Instad